mne/epochs.py

```diff
@@ -498,15 +503,44 @@ def plot(self, epoch_idx=None, picks=None, scalings=None,
             Whether to halt program execution until the figure is closed.
             Useful for rejecting bad trials on the fly by clicking on a
             sub plot.
+        n_epochs : int
+            The number of epochs to show on a continuous plot. If trellis is
```

Comment:
Let's just replace `continuous plot` everywhere with `mne_browse_epochs`. I feel that's more intuitive.